torch/csrc/Exceptions.cpp

```diff
@@ -51,6 +52,14 @@ could not be completed because the input matrix is singular.",
       PyModule_AddObject(module, "_LinAlgError", THPException_LinAlgError) ==
       0);
 
+  ASSERT_TRUE(
+      THPException_CUDAOutOfMemoryError = PyErr_NewException(
+          "torch.cuda.CUDAOutOfMemoryError", nullptr, nullptr));
```

Comment:
Should this inherit from RuntimeError or is the default Exception ok?

Reply:
For bc it probably should.